### PR TITLE
Notification banners

### PR DESCRIPTION
**Pull request by Rhian**
validation

### Description of change

We are adding two notification banners to the Planning Application Show view. One is a warning banner that displays if there are overdue validation requests that have not been responded to, and the other one is an information banner that shows validation requests that have been responded to.

### Story Link

https://trello.com/c/DKaQY1c7/346-add-notification-banner-when-change-requests-are-updated

### CODE REVIEW

File: app/helpers/validation_request_helper.rb [hunk 1]
```diff
@@ -38,4 +38,12 @@ def display_request_status(validation_request)
...
     end
   end
+
+  def overdue_requests(validation_requests)
+    validation_requests.select { |req| req.overdue? && req.state == "open" }
+  end
+
+  def closed_requests(validation_requests)
+    validation_requests.select { |req| req.state == "closed" }
+  end
```
Comment (james) [inline app/helpers/validation_request_helper.rb:48]: I think these two methods make more sense on the planning_application model than a helper.

File: app/views/planning_applications/_assessment_dashboard.html.erb [hunk 2]
```diff
@@ -8,6 +8,13 @@
 
 <%= render "proposal_header" %>
 
+<div class="govuk-grid-column-two-thirds">
+  <% if @planning_application.validation_requests %>
+    <%= render "overdue_banner" unless overdue_requests(@planning_application.validation_requests).empty? %>
+    <%= render "response_banner" unless closed_requests(@planning_application.validation_requests).empty? %>
```
Comment (james) [inline app/views/planning_applications/_assessment_dashboard.html.erb:14]: minor tweak, but I prefer `if .present?` as it's not a double negative.

**Review by james — APPROVED** [review]
Looks good! Two non-blocking refactor suggestions, so feel free to do them before merging, or plough on ahead :-)

File: app/views/planning_applications/_assessment_dashboard.html.erb (hunk 2 above)
Comment (Rhian) [reply app/views/planning_applications/_assessment_dashboard.html.erb]: Fixed!

**Pull request merged**